src/wallet/api/walletApi.ts

```diff
@@ -237,6 +239,14 @@ const walletApi = {
     });
   },
 
+  _txNotFoundGuard(data: unknown) {
```

Comment:
Maybe this should also be a static like in `HathorWallet` or move this to the error file like `WalletFromXPubGuard`

Reply:
It can't be static because `walletApi` is an object and `WalletFromXPubGuard` is an error type, right?